scripts/benchmark_parser/parse_log.rb

```diff
@@ -132,6 +139,42 @@ def remove_units(hash)
 def split_slash_keys(hash)
 end
 
+def get_test_code_commit
+  return 'NOT FOUND' if ENV['WORKSPACE'].nil?
+  current_directory = Dir.pwd
+  Dir.chdir ENV['WORKSPACE']
+  git_log = `git log -1`
+  Dir.chdir current_directory
+  return 'NOT FOUND' if git_log.nil?
+  commit_regex = /commit\s+(.+)/
+  if git_log.lines.first =~ commit_regex
+    return git_log.lines.first.match(commit_regex).captures[0]
+  end
+  return 'NOT FOUND'
+end
+
+def get_build_params_hash
+  template_path = ENV['name'] ? "#{ENV['name']}.json" : 'NOT FOUND'
+  cnf_path = File.exist?('maxscale.cnf') ? "#{Dir.pwd}/maxscale.cnf" : 'NOT FOUND'
+  return {
+      'jenkins_id' => ENV['BUILD_NUMBER'] || 'NOT FOUND',
```

Comment:
Use loop from build parser that will do the same (ENV['BUILD_NUMBER'] || 'NOT FOUND') action for all needed env variables.